pprof.graphqls

```diff
@@ -0,0 +1,174 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+# Request to create a pprof task
+input PprofTaskCreationRequest {
+    # Define the service to execute the task
+    serviceId: ID!
+    # Define which instances need to execute tasks
+    serviceInstanceIds: [String!]!
+    # Define the duration of this task
+    duration: Int!
```

Comment:
Following the agent implementation, do all event types require a duration? Also, please add the unit in the comment.